spiritualoutfitters.html

```diff
@@ -0,0 +1,209 @@
+            <div class="modal-body">
+              <iframe allowfullscreen="" src="https://www.youtube.com/embed/5iSAc8-1CVg" frameborder="0"></iframe>
+            </div>
+          </div>
+        </div>
+      </div>
+            </div>
+        </div>
+    </div>
+</section>
+<section class="bg-orange" style="background-image:url(https://images.ctfassets.net/y3a9myzsdjan/3O2BjsXNZXbqUMKvnOKA92/8813b233127074a3b029fc3aa8b83781/spiritual-outfitter-overlay.png);">
+    <div class="container-fluid">
+        <div class="row soft df-992">   
+            <div class="col-md-7 text-center push-ends soft-bottom ma-ends">
+                <img src="https://images.ctfassets.net/y3a9myzsdjan/5vNrrMcWsJGwWtxCfengjr/cbb5b329644926a113a65754c8bd2505/crossroads-logo-badge2.png">
+            </div>         
+                <div class="col-md-5 block bg-primary soft">
+                    <div class="soft-ends">
+                <h2 class="text-orange text-uppercase soft-top font-family-base-black mt-100">
+                    born<br>for<br>adventure
+                </h2>
+                <p class="font-family-serif soft-ends mb-100">
+                    “Faith” by its very nature is an exercise in adventure. The Bible is chock full of stories of people
+                    stepping into crazy adventures of faith. If this new look reminds you of Bear Grylls repelling off
+                    the Northface in Patagonia shorts that’s no accident. We’re not a “camping church”, but we are
+                    dedicated to risk, action, and movement. That doesn’t happen in a La-Z-Boy.
+                </p>   
+                </div>         
+            </div>           
+        </div>
+    </div>
+</section>
+<section style="background-color: #dbd1bf;">
+    <div class="container">
+        <div class="row soft-ends push-ends df-992">
+            <div class="col-md-6 soft-ends push-ends text-left">
+                <h1 class="text-primary text-uppercase soft-top font-family-base-black">
+                    the trail marker
+                </h1>
+                <p class="text-primary font-family-serif soft-ends">
+                    If you’re still reading this I’m guessing you’ve been around
+                    Crossroads for a while (or you have a logo fetish). For the past 14
+                    years, you’ve seen a little X in a circle icon thingy. We call it ‘the
+                    screwhead” because it looks like the top of a Phillips-head screw. You
+                    know it as the button you tap on your iPhone to close windows. Not
+                    exactly inspirational. We call the new X The Trail Marker. It’s bold
+                    and forward moving, just like us.
+                </p>
+            </div>
+            <div class="col-md-6 text-center ma-ends">
+            <img src="https://images.ctfassets.net/y3a9myzsdjan/gtgZUfgeUaKHymVo0jext/c1eb4ef9aa7b9f0d3d32889c9f87f54c/spiritual-outfitters-animation4__1_.gif">
```

Comment:
Use imgix url

Reply:
Thanks for catching this - we'll have to stick with using the ctfassets links because imgix doesn't work with the gifs.